common/hexutil/json.go

```diff
@@ -72,6 +72,25 @@ func (b Bytes) String() string {
 	return Encode(b)
 }
 
+// ImplementsGraphQLType returns true if Bytes implements the specified GraphQL type.
+func (b Bytes) ImplementsGraphQLType(name string) bool { return name == "Bytes" }
+
+// UnmarshalGraphQL unmarshals the provided GraphQL query data.
+func (b *Bytes) UnmarshalGraphQL(input interface{}) error {
+	var err error
+	switch input := input.(type) {
```

Comment:
prefer the syntax `input, ok := input.(type)` so that an error will be returned instead of panicking.

Reply:
I don't think that's possible here? This is a type switch, not a cast.

Reply:
right, my bad